src/swagger.js

```diff
@@ -54,7 +54,8 @@ const User = Joi.object({
   policies: Joi.array().items(PolicyRef)
 })
 
-const UserList = Joi.array().items(User)
+// not used, linter complaint.
```

Comment:
@p16 Couldn't push without resolving this. I'm happy to delete if it's not being used. Fails linter.